src/libcamera/utils.cpp

```diff
@@ -67,8 +67,8 @@ char *secure_getenv(const char *name)
 #if HAVE_SECURE_GETENV
 	return ::secure_getenv(name);
 #else
-	if (issetugid())
-		return NULL;
+//	if (issetugid())
```

Comment:
This should likely be wrapped better with a check that does "the right thing" if issetugid isn't present, and should have a define set for HAVE_SETUGID when it's available like the others.

It's fairly horrible code here anyway though (I mean before this patch)

Reply:
Modified to use HAVE_SETUGID